lib/Suggestions.js

```diff
@@ -67,8 +68,9 @@ class Suggestions extends Component {
 
   markIt = (input, query) => {
     const escapedRegex = query.trim().replace(/[-\\^$*+?.()|[\]{}]/g, '\\$&');
+    const html = input[this.props.labelField];
     return {
-      __html: input.text.replace(RegExp(escapedRegex, 'gi'), '<mark>$&</mark>'),
+      __html: html.replace(RegExp(escapedRegex, 'gi'), '<mark>$&</mark>'),
```

Comment:
labelField.replace

Reply:
I'm confused as to what you're wanting here and where you put
> ``` js 
> const { [this.props.labelField]: labelField } = input;
> ```

The labelField isn't what's being put into the `__html` field. Could you clarify for me?

Reply:
Yeah I get what it's doing, it was really the naming you put that was confusing. `labelFieldValue` is still a bit ambiguous to me..
What about 
```js
const { [this.props.labelField]: text } = input;
```
or 
```js
const { [this.props.labelField]: labelValue} = input;
```

It's really just semantics, is should just be clear to anyone what the actual value is.

Reply:
Yeah sorry my bad. `const { [this.props.labelField]: labelValue} = input;` this looks fine 👍

Reply:
this change seems to be pending